js/perf/codec-perf-rig/src/main.js

```diff
@@ -43,9 +43,9 @@ const args = argv
   .argv;
 
 main().catch(ex => {
-  console.error('\nError:', ex);
+  console.error('\nError:', ex);  // eslint-disable-line no-console
```

Comment:
do we still need these with the .eslintrc.js change at root of js/perf/ directory?